cmd/sharded-test-server/main.go

```diff
@@ -107,6 +109,11 @@ func start(proxyFlags, shardFlags []string, logDirPath, workDirPath string, numb
 		return err
 	}
 
+	standaloneVW := sets.NewString(shardFlags...).Has("--run-virtual-workspaces=false")
+	if standaloneVW {
+		shardFlags = append(shardFlags, fmt.Sprintf("--shard-virtual-workspace-url=https://%s:7444", hostIP))
```

Comment:
Wouldn't we run one per shard?

Reply:
Yeah that's probably a change I should make

Reply:
yes, one per shard please cc @p0lyn0mial